### PR TITLE
merge papilledema

### LINKED ISSUE #7279: MONDO:0002003 & MONDO:0006879 [Merge]
**Mondo term (ID and Label)**
MONDO:0002003 papilledema

**Reason for deprecation**
These terms may be synonyms? Based on overlapping Syns in Mondo and textual definitions these terms seem very similar.
Shared synonyms: choked disc; optic (disc) edema 

Optic papillitis: Swelling of the optic disk, usually in association with increased intracranial pressure, characterized by hyperemia, blurring of the disk margins, microhemorrhages, blind spot enlargement, and engorgement of retinal veins. Chronic papilledema may cause optic atrophy and visual loss. (Miller et al., Clinical Neuro-Ophthalmology, 4th ed, p175)

papilledema: Swelling around the optic disk, usually due to increased intracranial pressure or pressure on the optic nerve by a tumor.

**Term to be merged with**
MONDO:0006879 optic papillitis

**Your nano-attribution (ORCID)**
If you don't have an ORCID, you can sign up for one [here](https://orcid.org/)

### PR DESCRIPTION
**Pull request by twhetzel**
closes #7279

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -132346,50 +132319,73 @@ synonym: "optic nerve papillitis" EXACT [MESH:D010211]
...
 synonym: "papilledema associated with increased intracranial pressure" EXACT [MESH:D010211]
 synonym: "papillitis" EXACT [DOID:10175, MESH:D010211]
 synonym: "papillitis, optic" EXACT [MESH:D010211]
 synonym: "papillitis, optic nerve" EXACT [MESH:D010211]
+synonym: "papilloedema" EXACT [NCIT:C3307]
 synonym: "retinal edema" RELATED [MESH:D010211]
 synonym: "retinal Edemas" RELATED [MESH:D010211]
 synonym: "retinal oedema" RELATED OMO:0003005 []
 xref: DOID:10175 {source="EFO:1001074", source="MONDO:equivalentTo"}
+xref: DOID:146 {source="MONDO:equivalentTo"}
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:132333]: There are 2 DOID equivalent. If they are both valid, one should have the a MONDO:preferredExternal source

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -132346,50 +132319,73 @@ synonym: "optic nerve papillitis" EXACT [MESH:D010211]
...
 xref: EFO:1001074 {source="MONDO:equivalentTo", source="MONDO:EFO"}
 xref: ICD10CM:H35.81 {source="DOID:10175"}
 xref: ICD10CM:H46.0 {source="EFO:1001074", source="DOID:10175"}
+xref: ICD10CM:H46.0 {source="DOID:146"}
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:132337]: These ICD10CM are duplicated. Sources should be grouped. I think a normalization step should take care of this.

File: src/ontology/mondo-edit.obo [hunk 3]
```diff
@@ -132346,50 +132319,73 @@ synonym: "optic nerve papillitis" EXACT [MESH:D010211]
...
 xref: ICD10CM:H47.1 {source="DOID:10175"}
+xref: ICD10CM:H47.1 {source="DOID:146"}
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:132339]: These ICD10CM are duplicated. Sources should be grouped. I think a normalization step should take care of this.

File: src/ontology/mondo-edit.obo [hunk 4]
```diff
@@ -132346,50 +132319,73 @@ synonym: "optic nerve papillitis" EXACT [MESH:D010211]
...
 xref: ICD10CM:H47.10 {source="DOID:10175"}
+xref: ICD10CM:H47.10 {source="DOID:146"}
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:132341]: These ICD10CM are duplicated. Sources should be grouped. I think a normalization step should take care of this.

File: src/ontology/mondo-edit.obo [hunk 5]
```diff
@@ -132346,50 +132319,73 @@ synonym: "optic nerve papillitis" EXACT [MESH:D010211]
...
 xref: ICD10CM:H47.11 {source="DOID:10175"}
 xref: ICD9:362.83 {source="DOID:10175", source="MONDO:directSiblingOf"}
 xref: ICD9:377.0 {source="DOID:10175"}
+xref: ICD9:377.0 {source="DOID:146"}
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:132345]: same as above

File: src/ontology/mondo-edit.obo [hunk 6]
```diff
@@ -132346,50 +132319,73 @@ synonym: "optic nerve papillitis" EXACT [MESH:D010211]
...
 xref: ICD9:377.00 {source="MONDO:relatedTo", source="DOID:10175"}
+xref: ICD9:377.00 {source="MONDO:equivalentTo", source="MONDO:i2s", source="DOID:146"}
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:132347]: same as above

File: src/ontology/mondo-edit.obo [hunk 7]
```diff
@@ -132346,50 +132319,73 @@ synonym: "optic nerve papillitis" EXACT [MESH:D010211]
...
 xref: ICD9:377.01 {source="DOID:10175"}
+xref: ICD9:377.31 {source="MONDO:relatedTo", source="DOID:146"}
 xref: ICD9:377.31 {source="EFO:1001074", source="MONDO:equivalentTo", source="MONDO:i2s", source="DOID:10175"}
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:132350]: same as above

File: src/ontology/mondo-edit.obo [hunk 8]
```diff
@@ -132346,50 +132319,73 @@ synonym: "optic nerve papillitis" EXACT [MESH:D010211]
...
 xref: MedDRA:10030948 {source="EFO:1001074"}
+xref: MESH:D010211 {source="https://orcid.org/0000-0003-1967-3726", source="MONDO:equivalentTo", source="DOID:146"}
 xref: MESH:D010211 {source="EFO:1001074", source="DOID:10175"}
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:132353]: duplicated x-ref.

File: src/ontology/mondo-edit.obo [hunk 9]
```diff
@@ -132346,50 +132319,73 @@ synonym: "optic nerve papillitis" EXACT [MESH:D010211]
...
 xref: NCIT:C3307 {source="MONDO:otherHierarchy", source="DOID:10175"}
+xref: NCIT:C3307 {source="MONDO:equivalentTo", source="MONDO:otherHierarchy", source="MONDO:exact-label-match", source="DOID:146"}
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:132355]: duplicated x-ref. 
check which mondo source is correct: "equivalentTo" and "otherHierarchy" don't make sense together.

File: src/ontology/mondo-edit.obo [hunk 10]
```diff
@@ -132346,50 +132319,73 @@ synonym: "optic nerve papillitis" EXACT [MESH:D010211]
...
 xref: SCTID:111526001 {source="DOID:10175"}
+xref: SCTID:111526001 {source="DOID:146"}
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:132357]: duplicated x-ref.

File: src/ontology/mondo-edit.obo [hunk 11]
```diff
@@ -132346,50 +132319,73 @@ synonym: "optic nerve papillitis" EXACT [MESH:D010211]
...
 xref: SCTID:155187009 {source="DOID:10175"}
+xref: SCTID:155187009 {source="DOID:146"}
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:132359]: duplicated x-ref.

File: src/ontology/mondo-edit.obo [hunk 12]
```diff
@@ -132346,50 +132319,73 @@ synonym: "optic nerve papillitis" EXACT [MESH:D010211]
...
 xref: SCTID:193426002 {source="DOID:10175"}
 xref: SCTID:194038006 {source="DOID:10175"}
+xref: SCTID:194038006 {source="DOID:146"}
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:132362]: duplicated x-ref.

File: src/ontology/mondo-edit.obo [hunk 13]
```diff
@@ -132346,50 +132319,73 @@ synonym: "optic nerve papillitis" EXACT [MESH:D010211]
...
 xref: SCTID:194041002 {source="DOID:10175"}
+xref: SCTID:194041002 {source="DOID:146"}
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:132364]: duplicated x-ref.

File: src/ontology/mondo-edit.obo [hunk 14]
```diff
@@ -132346,50 +132319,73 @@ synonym: "optic nerve papillitis" EXACT [MESH:D010211]
...
+xref: SCTID:248487006 {source="DOID:146"}
 xref: SCTID:248487006 {source="DOID:10175"}
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:132366]: duplicated x-ref.

File: src/ontology/mondo-edit.obo [hunk 15]
```diff
@@ -132346,50 +132319,73 @@ synonym: "optic nerve papillitis" EXACT [MESH:D010211]
...
 xref: SCTID:3170006 {source="DOID:10175"}
 xref: SCTID:423341008 {source="MONDO:relatedTo", source="DOID:10175"}
+xref: SCTID:423341008 {source="MONDO:equivalentTo", source="DOID:146"}
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:132369]: duplicated x-ref.  Make sure you check which source is correct: equivalentTo or relatedTo.

File: src/ontology/mondo-edit.obo [hunk 16]
```diff
@@ -132346,50 +132319,73 @@ synonym: "optic nerve papillitis" EXACT [MESH:D010211]
...
 xref: SCTID:423488006 {source="DOID:10175"}
+xref: SCTID:423488006 {source="DOID:146"}
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:132371]: duplicated x-ref.

File: src/ontology/mondo-edit.obo [hunk 17]
```diff
@@ -132346,50 +132319,73 @@ synonym: "optic nerve papillitis" EXACT [MESH:D010211]
...
 xref: SCTID:6141006 {source="DOID:10175", source="MONDO:directSiblingOf"}
 xref: SCTID:73221001 {source="EFO:1001074", source="MONDO:equivalentTo", source="DOID:10175"}
+xref: SCTID:73221001 {source="MONDO:relatedTo", source="DOID:146"}
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:132374]: duplicated x-ref. Make sure you check which source is correct: equivalentTo or relatedTo.

File: src/ontology/mondo-edit.obo [hunk 18]
```diff
@@ -132346,50 +132319,73 @@ synonym: "optic nerve papillitis" EXACT [MESH:D010211]
...
 xref: UMLS:C0030353 {source="MONDO:notFoundInDiseaseSubset", source="DOID:10175"}
+xref: UMLS:C0030353 {source="MONDO:notFoundInDiseaseSubset", source="DOID:146"}
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:132376]: duplicated x-ref

**Review by sabrinatoro — CHANGES_REQUESTED** [review]
@twhetzel there are a lot of duplicated x-ref. I think that a normalization step will group them together. Please review the "source" annotation after that step though.

File: src/ontology/mondo-edit.obo (hunk 2 above)
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:132337]: done

File: src/ontology/mondo-edit.obo (hunk 3 above)
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:132339]: done

File: src/ontology/mondo-edit.obo (hunk 4 above)
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:132341]: done

File: src/ontology/mondo-edit.obo (hunk 5 above)
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:132345]: done

File: src/ontology/mondo-edit.obo (hunk 6 above)
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:132347]: done

File: src/ontology/mondo-edit.obo (hunk 7 above)
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:132350]: done

File: src/ontology/mondo-edit.obo (hunk 8 above)
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:132353]: fixed

File: src/ontology/mondo-edit.obo (hunk 9 above)
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:132355]: Agree, I can not find docs for "otherHierarchy"

File: src/ontology/mondo-edit.obo (hunk 10 above)
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:132357]: fixed

File: src/ontology/mondo-edit.obo (hunk 11 above)
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:132359]: fixed

File: src/ontology/mondo-edit.obo (hunk 12 above)
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:132362]: fixed

File: src/ontology/mondo-edit.obo (hunk 13 above)
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:132364]: fixed

File: src/ontology/mondo-edit.obo (hunk 14 above)
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:132366]: fixed

File: src/ontology/mondo-edit.obo (hunk 15 above)
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:132369]: updated to `xref: SCTID:423341008 {source="MONDO:relatedTo", source="DOID:146"}`

File: src/ontology/mondo-edit.obo (hunk 16 above)
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:132371]: fixed

File: src/ontology/mondo-edit.obo (hunk 17 above)
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:132374]: updated

File: src/ontology/mondo-edit.obo (hunk 18 above)
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:132376]: fixed

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:132333]: updated to 
```
xref: DOID:10175 {source="EFO:1001074", source="MONDO:equivalentTo"}
xref: DOID:146 {source="MONDO:relatedTo"}
```

**Comment by sabrinatoro** [comment]
@twhetzel for whatever reason, I cannot open "Files Changed" in GH and add my comment there. So I am adding it here
NCIT:C3307 has both "MONDO:equivalentTo" and "MONDO:otherHierarchy"

Screenshot 2024-05-31 at 12 46 58 PM


"MONDO:otherHierarchy" means that NCIT:C3307 is not in the "disease" branch. 
Therefore, "MONDO:equivalentTo" and "MONDO:otherHierarchy" are contradictory. 
Please remove the appropriate one

**Comment by sabrinatoro** [comment]
@twhetzel for whatever reason, I am getting the angry unicorn when I go to "files changed" (the page takes too long to load). And unfortunately, I need to go the "files changed" in order to approve. So I am stuck.
BUT I was able to review the changes on GH desktop, and I approve this PR. 
Please feel free to merge by bypassing the requirements to be met. (unless you know how to solve the unicorn problem which is only with this PR)

**Pull request merged**